Add description to README and minor fixes (#64)
tarantool/ddl: README.md

DDL module for Tarantool 1.10+

> The DDL module allows you to describe a data schema in the YAML format in a
> declarative way. It's alternative way to describe data schema using Lua,
> it is simpler to use, and you do not have to dive deep into Lua.

Reviewer: ```suggestion
The DDL module allows you to describe a data schema in the YAML format in a
declarative way. It's an alternative way to specify the schema rather than in
Lua. It's simpler to use, and you do not have to dive deep into Lua.
```

Reviewer 2: The DDL module enables you to describe data schema in a declarative YAML-based format.
It is a simpler alternative to describing data schema in Lua and doesn't require having a deep knowledge of Lua.

> DDL is a built-in Cartridge module. See more details about Tarantool's data mode
> in [documementation](https://www.tarantool.io/en/doc/latest/book/box/data_model/#data-schema-description-using-the-ddl-module).

Reviewer 2: ```suggestion
DDL is a build-in Cartridge module.
For more details about Tarantool's data model, refer to
[the documentation](https://www.tarantool.io/en/doc/latest/book/box/data_model/#data-schema-description-using-the-ddl-module).
```

Author: Thanks, Nick! The suggestion is committed.
